.prettierrc.mjs

```diff
@@ -0,0 +1,9 @@
+/**
+ * @type {import('prettier').Options}
+ */
+export default {
+  semi: false,
+  singleQuote: true,
+  arrowParens: 'avoid',
+  plugins: ['prettier-plugin-tailwindcss'],
+}
```

Comment:
Why did you change te file format? Also don't you need to pass `tailwindFunctions` anymore?

Reply:
I suspect due to switch from yarn to pnpm, it was not working with old format and different settings.

Reply:
🤨🤨🤨

Reply:
[`4f7f155` (#641)](https://github.com/status-im/status-web/pull/641/commits/4f7f15501f29fadf2195e331a911123e0efda609)